hyperplane/item.py

```diff
@@ -120,14 +120,17 @@ def __file_thumb_done(self, failed: bool) -> None:
         self.icon.set_visible(failed)
         self.picture.set_visible(not failed)
 
+        self.icon.set_css_classes(["large-icons"])
+        self.thumbnail.set_css_classes(["item-thumbnail"])
+        self.extension_label.set_css_classes(["file-extension"])
+
         if failed:
             self.icon.add_css_class(self.color + "-icon")
             self.thumbnail.add_css_class(self.color + "-background")
             self.extension_label.add_css_class(self.color + "-extension")
-            return
-
-        self.thumbnail.add_css_class("gray-background")
-        self.extension_label.add_css_class(self.color + "-extension-thumb")
+        else:
```

Comment:
Is there a reason to use else instead of return?

Reply:
I think it's just simpler and cleaner in cases like this, simply because of the semantics, e.g. *"if something then do this else do that"*, and not *"if something then do this and return, but otherwise there's more code further down"*. But that's it.

Reply:
I just like to avoid indentation as much as possible, I think it makes the code cleaner to read, so I prefer this syntax at places like this.

Reply:
Hmm, ok. That also makes sense.